### DISCUSSION TITLE
error "405 not allowed"

### DISCUSSION THREAD

**Discussion by Ahmed-Bawza** [community/community#125681] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'm building a web using github pages. my web using openCV library. when i reach the point that the web should use the cv2 functions, it show the error page. i don't know how to set the opencv environment or solve the problem
<img width="893" alt="Screenshot 2024-05-25 at 3 06 02 PM" src="https://github.com/community/community/assets/96870301/d6130e58-3234-46f2-8c32-f7bae28db9d2">
<img width="893" alt="Screenshot 2024-05-25 at 3 06 09 PM" src="https://github.com/community/community/assets/96870301/bcd2f87b-3e91-422a-917c-b54a3a9cfa90">

i have html , python file include flask use and yml file

**Comment by pedrolemos4** [comment]
Did you implement the method in Flask? Maybe it's a GET or POST that is missing if it is an API

**Reply by Ahmed-Bawza** [reply]
How can i check?

**Reply by pedrolemos4** [reply]
What methods in the API are you trying to access in that window? Check if all of them are implemented in the API and that you are using the correct verb (GET, POST, PUT, DELETE, etc) in the API
Usually 405 means that you did not implement that method
It may also help to use Developer Tools in your browser (Ctrl+Shift+I) and check the network tab

**Reply by Ahmed-Bawza** [reply]
I’m not that familiar with web develop but i’ll try, thx btw

**Reply by pedrolemos4** [reply]
You can maybe put the scripts here and I can give it a quick look.
If you were able to fix it please mark my response as correct, thanks :)

**Reply by Ahmed-Bawza** [reply]
You can check the repo:
https://github.com/Ahmed-Bawza/areaWeb.git

**Reply by pedrolemos4** [reply]
I don't know the code...
In the page that you are do what I told you, go to dev tools and refresh the page when you are in the networking tab, and put a print here

**Reply by pedrolemos4** [reply]
You are defining the script but you are specifying the wrong path
You have 'js/script.js'
And should have '../js/script.js'

**Comment by 1saptarshi** [comment]
I think it's a GET or POST that is missing if it is an API

**Comment by 1saptarshi** [comment]
> I think it's a GET or POST that is missing if it is an API ??

**Reply by Ahmed-Bawza** [reply]
Don’t know what you mean
